.github/dependabot.yml

```diff
@@ -5,7 +5,11 @@ updates:
     schedule:
       interval: "daily"
```

Comment:
maybe we need to reduce this to weekly so that it can actually be grouped?

Reply:
I was hoping it would update existing group PRs and continously keep a single PR up to date.